### PR TITLE
wm _html files

### PR DESCRIPTION
**Pull request by whitney-mitchell**
Hey guys! I'm open to any changes here.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			<div class="navLinks"><a href="products.html" title="Our_Bikes">Our Bikes</a></div>
+		</nav>
+	</header>
+
+
+<!-- HOMEPAGE CONTENT -->
+
+	<div class="mainContent">
+		<div id="splashPicture">
+			<img src="images/bikeRide.jpg" alt="Vroom.">
```
Comment (josephkane) [inline index.html:29]: I have a folder named "img" with all the product images in it, maybe you can slack this file to me tomorrow and i'll add it in?

EDIT: nevermind, you can do that on your local drive once both our PRs are pulled in, right? sorry, i'm a total spaz. Nice job!

**Comment by josephkane** [comment]
Looks great!

:+1:

File: index.html (hunk 1 above)
Comment (whitney-mitchell) [inline index.html:29]: Whoops, I forgot to mention that here and in contact.html, the images are in no way real. Just placeholders.

**Comment by DamonRomano** [comment]
:+1:

**Pull request merged**